nevergrad/functions/multiobjective.py

```diff
@@ -0,0 +1,387 @@
+#    This file is part of DEAP.
+#
+#    Copyright (C) 2010 Simon Wessing
+#    TU Dortmund University
+#
+#    In personal communication, the original authors authorized DEAP team
+#    to use this file under the Lesser General Public License.
+#
+#    You can find the original library here :
+#    http://ls11-www.cs.uni-dortmund.de/_media/rudolph/hypervolume/hv_python.zip
+#
+#    DEAP is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Lesser General Public License as
+#    published by the Free Software Foundation, either version 3 of
+#    the License, or (at your option) any later version.
+#
+#    DEAP is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Lesser General Public License for more details.
+#
+#    You should have received a copy of the GNU Lesser General Public
+#    License along with DEAP. If not, see <http://www.gnu.org/licenses/>.
+#
+#    Modified by O. Teytaud as needed for Nevergrad.
+
+from ..instrumentation import InstrumentedFunction
+from math import log, floor
+import random
+from typing import (Tuple, Any, Callable, List, Optional, Dict, ValuesView, Iterator,
+                    TypeVar, Generic, Deque, Iterable)
+import warnings
+import numpy
+#from ..optimization import base
+from ..optimization.base import Candidate
+from ..common.typetools import ArrayLike
+from ..instrumentation import InstrumentedFunction
+
+def hypervolume(pointset, ref):
```

Comment:
anything that you have added should be mode to a different file:
I would go with a directiory:
multiobjective
   __init__.py
   hypercube.py (or the initial name of the file)
   core.py (our file)

then we'll add files with examples